src/bridge/BridgeSettlerBase.sol

```diff
@@ -19,6 +19,7 @@ abstract contract BridgeSettlerBase is Basic {
     using FastDeployer for IDeployer;
 
     event GitCommit(bytes20 indexed);
+    error CounterfeitSettler(address counterfeitSettler);
```

Comment:
Consider adding this to `SettlerErrors.sol` or making a new file for BridgeSettler errors